provider_api/locations.md

```diff
@@ -346,6 +346,7 @@ Check the list below to see details for all fields.
 | `area_to`                             | Integer                               | Maximum area in m2. Must be more than area_from. | Optional
 | `description`                         | String                                | A detailed description of the location. Use \n for new line. | Optional
 | `energy_rating`                       | String                                | "A","B","C","D","E","F" or "G". null to remove. | Optional
+| `exempted_from_energy_rating`         | Boolean                               | Indicate the location is exempted from having energy rating. Default is false. | Optional
```

Comment:
Should it say something about the fact that if this is true, then `energy_rating` is nilled?